cloud_functions/climateiq_data_export_cf/main_test.py

```diff
@@ -19,16 +19,16 @@ def test_export_model_predictions_invalid_object_name() -> None:
         "bucket": "climateiq-predictions",
         "name": "invalid_name",  # Invalid object name
     }
-    event = CloudEvent(attributes, data)
+    event = http.CloudEvent(attributes, data)
 
     with pytest.raises(ValueError) as exc_info:
         main.export_model_predictions(event)
 
     assert "Invalid object name format. Expected 5 components." in str(exc_info.value)
 
 
-@mock.patch.object(main.storage, "Client", autospec=True)
-@mock.patch.object(main.firestore, "Client", autospec=True)
+@mock.patch.object(storage, "Client", autospec=True)
```

Comment:
Why the change here? These should be the equivalent because `storage` is a singleton. I generally prefer the former `@mock.patch.object(main.firestore, "Client", autospec=True)` because the important thing is that you're patching the objects used by the functions inside the `main` module that you're testing.

Reply:
It's a style thing but I've always named the mocked object directly instead of indirectly through the tested module's dependencies. I don't think I've seen this style within google3 before but it's been a few years since I wrote Python extensively. I personally find this more readable because it's easier to understand where the dependencies come from. But if the rest of the ClimateIQ code names the dependencies indirectly, then I can change this.